cypress/fixtures/docs/dynamicObjects.md

```diff
@@ -0,0 +1,275 @@
+    setContent: ...,
+    event: ...,
+    eventValidationJsonPath: ...,
+    content: ...
+  };
+```
+
+| **Param**               | **Definition**                                                                                            | **Example**                                   | **Supported Type**             |
+| ----------------------- | --------------------------------------------------------------------------------------------------------- | --------------------------------------------- | ------------------------------ |
+| method                  | The method name to make a call, associated with get test steps                                            | accessibility.closedCaptionsSettings          | string                         |
+| params                  | parameters of the method                                                                                  | {}                                            | string or object               |
+| validationJsonPath      | Contains an array or string with the path of the response value that needs to be validated                | result.enabled                                | string or array                |
+| setMethod               | The setter method name to make a call, associated with set test steps                                     | closedcaptions.setEnabled                     | string                         |
+| setParams               | Parameters of the set method to set new value.                                                            | {value: true}                                 | string or object               |
+| setValidationJsonPath   | Contains an array or string with the path of the response value that needs to be validated.               | result.enabled                                | string or array                |
+| setContent              | Holds the source of truth to validate against the response of the set method.                             | null                                          | string, null, boolean or array |
+| event                   | Name of the event to start listening for.                                                                 | accessibility.onClosedCaptionsSettingsChanged | string                         |
+| eventValidationJsonPath | Contains an array or string with the path of the response value that needs to be validated for the event. | result.enabled                                | string or array                |
+| content                 | Holds the source of truth to validate against the response of th get method or event.                     | true                                          | string, null, boolean or array |
+
+**Note:**
+
+- Acceptable types for the above params i.e string, number, array, object etc.
+- These are all the supported fields but depending on the test scenario and glue code used none of them are technically required.
+
+## Runtime Variables
```

Comment:
Replace line 60 with 
Runtime variables are used to store values from feature file and use them in dynamic firebolt object for making api calls, api validation, event validation etc. 

Line 62 to be repleced with - Below are the runtime variables that is used by fireboltCall object. 

Line 67 to be replaced with - **Note:** `resolveAtRuntime()` function is used by fireboltCall object in order to process runtime variables and other fields.

Reply:
done